.github/workflows/versionControlAndAuditCheck.yml

```diff
@@ -80,7 +80,7 @@ jobs:
             fi
           done <<< "${FILES}"
 
-          ##### if none found, exit here as there is nothing to do
+          ##### if nothing found, exit here as there is nothing to do
```

Comment:
This was required to silence the pre-commit checker as we use "none" as value for blockscout API key